docs/xdsl-introduction.ipynb

```diff
@@ -0,0 +1,937 @@
+{
```

Comment:
<p>This section needs a minor spell-checker pass. Maybe we could also revisit a few parts of the text, but overall a good start I think.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xdslproject/xdsl/pull/398/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>spell checked the notebook!</p>